### PR TITLE
Support VK_NV_device_generated_commands in addition to the old NVX variant

### LINKED ISSUE #12: Build error  (undefined *_NVX symbols) with Vulkan SDK 1.2.135.0
`VK_PIPELINE_STAGE_COMMAND_PROCESS_BIT_NVX`, `VK_ACCESS_COMMAND_PROCESS_READ_BIT_NVX`, and `VK_ACCESS_COMMAND_PROCESS_WRITE_BIT_NVX` are no longer defined in this week's Vulkan SDK. I naively assume they've morphed into `*_PREPROCESS_*_NV` but what do I know, I'm way out of the loop these days :)

### PR DESCRIPTION
**Pull request by cdwfs**
Fixes #12

### CODE REVIEW

File: thsvs_simpler_vulkan_synchronization.h [hunk 1]
```diff
@@ -182,9 +182,10 @@ typedef enum ThsvsAccessType {
     THSVS_ACCESS_NONE,                                                      // No access. Useful primarily for initialization
 
 // Read access
-    // Requires VK_NVX_device_generated_commands to be enabled
-    THSVS_ACCESS_COMMAND_BUFFER_READ_NVX,                                   // Command buffer read operation as defined by NVX_device_generated_commands
-
+#if defined(VK_NV_device_generated_commands) || defined(VK_NVX_device_generated_commands)
+    // Requires VK_NV_device_generated_commands to be enabled
+    THSVS_ACCESS_COMMAND_BUFFER_READ_NV,                                    // Command buffer read operation as defined by NV_device_generated_commands
+#endif
```
Comment (Tobski) [inline thsvs_simpler_vulkan_synchronization.h]: Why are there ifdefs around this? I don't ifdef anything else in here. I could agree to ifdeffing all of the extensions but just this one seems out of place.

**Review by Tobski — CHANGES_REQUESTED** [review]
I think given that the NVX extension isn't officially supported any more, we could probably just drop all references to NVX in here. Probably should've updated this a while ago 🤦

**Comment by Tobski** [comment]
(BTW, thanks for doing this - I had kind of forgotten this needed updating!)

File: thsvs_simpler_vulkan_synchronization.h (hunk 1 above)
Comment (cdwfs) [reply thsvs_simpler_vulkan_synchronization.h]: The goal was to continue to support older SDKs that expose the NVX extension but not the NV variant. If that's not a priority, I'd be totally fine just replacing NVX->NV unconditionally.

**Comment by Tobski** [comment]
Ah sorry didn't see you updated this. Accepting now!

**Review by Tobski — APPROVED** [review]

**Pull request merged**